chess/frontend/gui/src/App.js

```diff
@@ -1,26 +1,14 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
-
+import React, { Component } from "react";
+import { Route, Switch, BrowserRouter } from "react-router-dom";
+import User from "./modules/ViewUser/User";
```

Comment:
Is this just reading a direct path? Is that how things are normally done? Just asking, I don't know how this works

Reply:
Yeah I am going to add all the modules relative to this parent directory